### PR TITLE
chore: Update large blob test to use 16mb

### PR DESCRIPTION
**Pull request by epociask**
## Fixes Issue

Fixes #

### CODE REVIEW

File: store/router.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (r *Router) multiSourceRead(ctx context.Context, commitment []byte, fallbac
 			continue
 		}
 		// verify cert:data using EigenDA verification checks
-		err = r.eigenda.Verify(key, data)
+		err = r.eigenda.Verify(commitment, data)
```
Comment (bxue-l2) [inline store/router.go:192]: Is it true that the only cached source is eigenda?

File: e2e/server_test.go [hunk 2]
```diff
@@ -110,8 +110,8 @@ func TestProxyServerWithLargeBlob(t *testing.T) {
 		URL: ts.Address(),
 	}
 	daClient := client.New(cfg)
-	//  4MB blob
-	testPreimage := []byte(e2e.RandString(4_000_000))
+	//  16MB blob
+	testPreimage := []byte(e2e.RandString(16_000_000))
```
Comment (bxue-l2) [inline e2e/server_test.go:114]: eigenda accounting uses 2^24, 16777216

**Comment by bxue-l2** [comment]
If possible, can you also update the readme, and default argument, they are current 4MiB

File: store/router.go (hunk 1 above)
Comment (bxue-l2) [reply store/router.go:192]: I am not super familiar with multisource

**Review by bxue-l2 — APPROVED** [review]

File: e2e/server_test.go (hunk 2 above)
Comment (epociask) [reply e2e/server_test.go:114]: right - the default codec client will pad to the next power of two when doing dispersal

File: store/router.go (hunk 1 above)
Comment (epociask) [reply store/router.go:192]: nah the cached sourced can be arbitrary,  but we'll need to do the eigenda verifications against it to ensure the data hasn't been tampered with

**Pull request merged**